src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
I think the reason we report T2_CH_CERN PNN instead is because the logCollect tarball is also - optionally - staged to EOS, making it easier for anyone looking up for a PFN.

Having a hacky code does not mean you can make it worse. I'm totally against adding LogCollect to wmbs_location and over complicating what's already complicated. We better not have any report then (as we do for the last several years, without any issues).